src/components/Modeler.vue

```diff
@@ -789,26 +788,6 @@ export default {
       this.allowDrop = allowDrop;
       this.poolTarget = poolTarget;
     },
-    addStartEvent() {
-      /* Add an initial startEvent node if the graph is empty */
-      if (this.nodes.length > 0) {
-        return;
-      }
-
-      const definition = startEvent.definition(this.moddle, this.$t);
-      const diagram = startEvent.diagram(this.moddle);
-
-      diagram.bounds.x = 150;
-      diagram.bounds.y = 150;
-
-      this.addNode({
-        definition,
-        diagram,
-        type: startEvent.id,
-      });
-
-      setTimeout(() => undoRedoStore.dispatch('resetHistory'));
```

Comment:
Since it seems it was only used here, the `resetHistory` action could be removed from the undo/redo store.